docs/backend/README.md

```
@@ -0,0 +1,406 @@
+
+```bash
+php artisan typescript:transform
+```
+
+## Configuration Management
+
+### Environment-based Configuration
+```php
+// config/features.php
+return [
+	// ...
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)